draft/spec/index.html

```diff
@@ -1192,107 +1192,107 @@ <h2>Moab in an OCFL Object</h2>
+    "e15267...58d": [ "v0001/data/metadata/versionMetadata.xml" ],
+    "0d9e0b...9a2": [ "v0001/data/metadata/contentMetadata.xml" ],
+    "dd9289...31d": [ "v0001/data/metadata/relationshipMetadata.xml" ],
+    "f90947...11b": [ "v0002/manifests/manifestInventory.xml" ],
+    "bb3011...a58": [ "v0002/manifests/versionAdditions.xml" ],
+    "0dc4fc...f9f": [ "v0002/manifests/signatureCatalog.xml" ],
+    "2f1cf8...ea1": [ "v0002/manifests/fileInventoryDifference.xml" ],
+    "45b457...db5": [ "v0002/manifests/versionInventory.xml" ],
+    "7519c5...63f": [ "v0002/data/metadata/provenanceMetadata.xml" ],
+    "abda4c...622": [ "v0002/data/metadata/workflows.xml" ],
+    "76549e...b2b": [ "v0002/data/metadata/rightsMetadata.xml" ],
+    "bdc4d6...3b6": [ "v0002/data/metadata/events.xml" ],
+    "7b331c...f9b": [ "v0002/data/metadata/identityMetadata.xml" ],
+    "80ceac...b9c": [ "v0002/data/metadata/versionMetadata.xml" ],
+    "4853a2...fbe": [ "v0002/data/metadata/contentMetadata.xml" ],
+    "1d5090...f5f": [ "v0002/data/metadata/relationshipMetadata.xml" ],
+    "f209bf...ceb": [ "v0002/data/metadata/embargoMetadata.xml" ],
+    "04461b...5c6": [ "v0003/manifests/manifestInventory.xml" ],
+    "190103...20f": [ "v0003/manifests/versionAdditions.xml" ],
+    "24a84a...82d": [ "v0003/manifests/signatureCatalog.xml" ],
+    "5f4d52...7ef": [ "v0003/manifests/fileInventoryDifference.xml" ],
+    "2d7f8d...d67": [ "v0003/manifests/versionInventory.xml" ],
+    "dd9125...d4b": [ "v0003/data/metadata/technicalMetadata.xml" ],
+    "d9e177...477": [ "v0003/data/metadata/provenanceMetadata.xml" ],
+    "4f5908...4f5": [ "v0003/data/metadata/workflows.xml" ],
+    "e64db0...500": [ "v0003/data/metadata/descMetadata.xml" ],
+    "05fa51...818": [ "v0003/data/metadata/rightsMetadata.xml" ],
+    "d70dd8...5ad": [ "v0003/data/metadata/events.xml" ],
+    "509a2d...dc6": [ "v0003/data/metadata/identityMetadata.xml" ],
+    "548066...893": [ "v0003/data/metadata/versionMetadata.xml" ],
+    "93884e...aae": [ "v0003/data/metadata/contentMetadata.xml" ],
+    "4c5ab4...b02": [ "v0003/data/metadata/embargoMetadata.xml" ]
 }
 </pre>
     <p>
         The version 1 state block would look like this. Note the absence of the manifests/ directory.
     </p>
 <pre>
 "state": {
-	"98114a...588":	 ["data/content/eric-smith-dissertation-augmented.pdf"],
-	"7f3d87...15b":	 ["data/content/eric-smith-dissertation.pdf"],
-	"6d19f0...064":	 ["data/metadata/technicalMetadata.xml"],
-	"6e4be4...375":	 ["data/metadata/provenanceMetadata.xml"],
-	"d8a319...d0f":	 ["data/metadata/descMetadata.xml"],
-	"de823a...acc":	 ["data/metadata/rightsMetadata.xml"],
-	"080617...40c":	 ["data/metadata/identityMetadata.xml"],
-	"e15267...58d":	 ["data/metadata/versionMetadata.xml"],
-	"0d9e0b...9a2":	 ["data/metadata/contentMetadata.xml"],
-	"dd9289...31d":	 ["data/metadata/relationshipMetadata.xml"]
+    "98114a...588": ["data/content/eric-smith-dissertation-augmented.pdf" ],
```

Comment:
The block from lines 1246 through 1255 is missing space after opening `[`

Reply:
So they are! Fixed.